### DISCUSSION TITLE
Slicer parts with connectors

### DISCUSSION THREAD

**Discussion by rhaleluk** [OrcaSlicer/OrcaSlicer#1122] [General]
Bambu just did an update and added the ability to slice parts and add pegs / connectors to the parts. It's a great feature and so I was curious if there are plans to bring this functionality to Orca. 

Orca has been a game changer for us at Print Parts - we prefer to use it over the Bambu slicer. 

I'd love to have the feature to slice parts and add connectors to print larger models across various machines. 

Thanks!

**Comment by SoftFever** [comment]
@rhaleluk 
It's there in the 1.6.3 beta release already: https://github.com/SoftFever/OrcaSlicer/releases/tag/v1.6.3-beta

**Reply by rhaleluk** [reply]
Awesome! You are the man!

**Reply by rhaleluk** [reply]
My issue is that when you use the cut connectors and try to export the STL file, the side with the holes for the plugs doesn't export with the holes. You can export the side with the plugs and the plugs are in the updated cut STL file, but the adjacent side with the holes that line up with the plugs doesn't export with the holes cutout. Can that be fixed?

**Reply by liftbag** [reply]
> My issue is that when you use the cut connectors and try to export the STL file, the side with the holes for the plugs doesn't export with the holes. You can export the side with the plugs and the plugs are in the updated cut STL file, but the adjacent side with the holes that line up with the plugs doesn't export with the holes cutout. Can that be fixed?

This is how PrusaSlicer works too. Bambu and Orca merge PrusaSlicer's new features and most of the time they work exactly like in PrusaSlicer. I don't know the reason to export the part with the holes in stl as if they weren't there. Could it be for copyright reasons?

**Reply by SoftFever** [reply]
These holes are essentially `negative part`, that is the reason the exported stl files don't contain them
